tests/integration/tests/charm_tests/openstack_exporter.py

```diff
@@ -161,8 +161,14 @@ def test_configure_ssl_ca(self):
         results = model.run_on_leader(APP_NAME, command)
         clouds_yaml = results.get("Stdout", "").strip()
         data = yaml.safe_load(clouds_yaml)
-        cacert = data["clouds"][CLOUD_NAME]["cacert"]
-        self.assertEqual(cacert, f"{new_value}")
+        cacert_path = data["clouds"][CLOUD_NAME]["cacert"]
+        self.assertEqual(cacert_path, OS_CLIENT_CONFIG_CACERT)
+
+        # Verify ssl_ca was written to the file
+        command = f"sudo cat {cacert_path}"
```

Comment:
I think it's better to use `model.block_until_file_has_contents(APP_NAME, cacert_path, new_value)`